docs/CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ Entries inside each section should be ordered by type:
 
 # unreleased - YYYY-MM-DD
 ## Python API
+* [Removed] Python 3.7 support ([#3841](https://github.com/quiltdata/quilt/pull/3841))
```

Comment:
```suggestion
* [Removed] Drop Python 3.7 support ([#3841](https://github.com/quiltdata/quilt/pull/3841))
```

Reply:
actually why?
it actually was "removed Python 3.7 support"

Reply:
tag is not a part of the sentence, the sentence should be complete without a tag and have imperative form